lib/elixir/lib/module.ex

```diff
@@ -409,7 +409,7 @@ defmodule Module do
     * `:module`     - module name (`Module == Module.__info__(:module)`)
 
   In addition to the above, you may also pass to `__info__/1` any atom supported
-  by `:erlang.module_info/0` which also gets defined for each compiled module.
+  by `:erlang.module_info/0` [should link to http://erlang.org/doc/reference_manual/modules.html#id77914 ] which also gets defined for each compiled module.
```

Comment:
Please use:

    [`:erlang.module_info/0`](http://erlang.org/doc/reference_manual/modules.html#id77914)

And thanks!